test/index.js

```diff
@@ -278,34 +278,6 @@ describe('koa-body', async () => {
     });
   });
 
-  describe('onError option', () => {
-    it('onError should not capture errors thrown in downstream middleware', async () => {
```

Comment:
I think we should come up with another way to test this. I have an idea and can try it out in a few hours.